### PR TITLE
Clarify networking stack v1 and v2 environment variables

### PR DESCRIPTION
**Pull request by dwightjl**

### CODE REVIEW

File: docs/chainlink-nodes/configuration-variables.md [hunk 1]
```diff
@@ -1560,6 +1562,12 @@ Example: `P2P_PEER_ID=12D3KooWMHMRLQkgPbFSYHwD3NBuwtS1AmxhvKVUrcfyaGDASR4U`
 
 ### Networking Stack V2
 
+> 🚧 If using the Networking Stack V2, you must unset the following Networking Stack V1 configuration variables:
+>
+> - [P2P_LISTEN_PORT](#p2p_listen_port)
```
Comment (chainchad) [inline docs/chainlink-nodes/configuration-variables.md:1567]: @jmank88 should we list all of the other networking stack v1 env vars here too?

Comment (dwightjl) [reply docs/chainlink-nodes/configuration-variables.md]: Added

File: docs/chainlink-nodes/configuration-variables.md [hunk 2]
```diff
@@ -1560,6 +1562,15 @@ Example: `P2P_PEER_ID=12D3KooWMHMRLQkgPbFSYHwD3NBuwtS1AmxhvKVUrcfyaGDASR4U`
 
 ### Networking Stack V2
 
+> 🚧 If using the Networking Stack V2, you must unset the [Networking Stack V1](#networking-stack-v1) configuration variables:
+>
+> - [P2P_ANNOUNCE_IP](#p2p_announce_ip)
```
Comment (chainchad) [inline docs/chainlink-nodes/configuration-variables.md:1567]: I just want to be certain that we should unset all of these other vars. I know for a fact that the following should be unset:

```
P2P_LISTEN_PORT
P2P_ANNOUNCE_PORT
P2P_ANNOUNCE_IP
```

But not sure if `P2P_BOOTSTRAP_PEERS` and `P2P_PEER_ID` should be not be used if using v2 networking stack.

File: docs/chainlink-nodes/configuration-variables.md [hunk 3]
```diff
@@ -1560,6 +1562,15 @@ Example: `P2P_PEER_ID=12D3KooWMHMRLQkgPbFSYHwD3NBuwtS1AmxhvKVUrcfyaGDASR4U`
...
+> - [P2P_ANNOUNCE_PORT](#p2p_announce_port)
+> - [P2P_BOOTSTRAP_PEERS](#p2p_bootstrap_peers)
+> - [P2P_LISTEN_IP](#p2p_listen_ip)
+> - [P2P_LISTEN_PORT](#p2p_listen_port)
+> - [P2P_PEER_ID](#p2p_peer_id)
```
Comment (jmank88) [inline docs/chainlink-nodes/configuration-variables.md:1572]: IIUC this one is shared between v1 and v2

Comment (dwightjl) [reply docs/chainlink-nodes/configuration-variables.md]: Resolved

File: docs/chainlink-nodes/configuration-variables.md (hunk 2 above)
Comment (dwightjl) [reply docs/chainlink-nodes/configuration-variables.md:1570]: Resolved based on @jmank88 's feedback.

File: docs/chainlink-nodes/configuration-variables.md (hunk 1 above)
Comment (dwightjl) [reply docs/chainlink-nodes/configuration-variables.md]: Fixed

**Review by chainchad — APPROVED** [review]

**Review by jmank88 — APPROVED** [review]

**Pull request merged**